### PR TITLE
Fix normalization indexing bug

### PR DESCRIPTION
**Pull request by streeve**
Also modifies unit test to use different error threshold per model

### CODE REVIEW

**Review by pzhanggit — APPROVED** [review]
Thanks!

**Comment by streeve** [comment]
Requested @allaffa to review for the error threshold changes

File: test_graphs.py [hunk 1]
```diff
@@ -36,12 +36,19 @@ def pytest_train_model(model_type):
 
     error, true_values, predicted_values = test_trained_model(tmp_file, model_type)
 
+    # Set RMSE and sample error thresholds
+    thresholds = {
```
Comment (pzhanggit) [inline test_graphs.py:40]: Is it supposed to be a temporary fix?

**Review by pzhanggit — COMMENTED** [review]
I am still a little hesitated to use different error thresholds for different models in the unit test. Maybe we can try to tune the structure and get similar accuracy?

File: test_graphs.py (hunk 1 above)
Comment (streeve) [reply test_graphs.py:40]: No, I think it's reasonable to have different thresholds per model

Comment (streeve) [reply test_graphs.py:40]: I would definitely expect to be able to improve GAT though

**Pull request merged**